### DISCUSSION TITLE
There is no way to Create a card token

### DISCUSSION THREAD

**Discussion by ziagit** [flutter-stripe/flutter_stripe#1356] [Q&A]
Hello there,

In this plugin there is no way to create a card token in flutter to send that token in the server for creating charges. 
If there is such feature please guide us someone to the process.

Thank you?

**Comment by Sahil-kachhap** [comment]
@ziagit Please try **createToken** method, and let me know if it does not works for your usecase. I will try to find other alternatives if it not works.

Here is its function signature:- 
`Future<TokenData> createToken(CreateTokenParams params) async {...}`
 this creates a single-use token that represents a credit card’s details.

**Reply by nglro** [reply]
Not sure what the use case for that method could be. It takes as parameters just a name and address, not any card details. It does not save a card and return a token that you can use to get that card details.

If you would call https://api.stripe.com/v1/tokens you would need to send:
```
card[name]	
card[number]	
card[exp_month]	
card[cvc]	
card[exp_year]	
```
and you would get back an object containing a token and the newly created card details.

https://stripe.com/docs/api/tokens/create_card

**Comment by HayaBekheet** [comment]
Did you find a solution to this problem? I'm stuck to it.
